src/openzaak/components/documenten/tests/test_objectinformatieobject.py

```diff
@@ -519,36 +555,83 @@ def test_list_filter_by_external_besluit(self):
         self.assertEqual(len(response.data), 1)
         self.assertEqual(response.data[0]["object"], besluit2)
 
-    def test_destroy_external_zaak(self):
-        zaak = "https://externe.catalogus.nl/api/v1/zaken/1c8e36be-338c-4c07-ac5e-1adf55bec04a"
+    @requests_mock.Mocker()
+    def test_destroy_external_zaak(self, m):
```

Comment:
Really minor remark, but I find that the name of the test doesn't to match well what happens in the test